views/index.erb

```diff
@@ -76,27 +76,37 @@
           </div>
         </div>
       <% else %>
-        <div class="usa-nav__secondary">
-          <ul class="usa-nav__secondary-links sign-in-type-wrap">
-            <li class="usa-nav__secondary-item">
-              <a href="?ial=1"><span class="<%= ial1_link_class %>">IAL1</span></a>
-            </li>
-            <li class="usa-nav__secondary-item">
-              <a href="?ial=2"><span class="<%= ial2_link_class %>">IAL2</span></a>
-            </li>
-            <li class="usa-nav__secondary-item">
-              <a href="?ial=0"><span class="<%= ialmax_link_class %>">IALMAX</span></a>
-            </li>
-          </ul>
-          <div class="sign-in-wrap">
-            <a class="usa-button usa-button--outline sign-in-bttn" href="<%= ial_url %>" style="float:right">
-              <svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-                <path fill="currentColor" d="M416 448h-84c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h84c17.7 0 32-14.3 32-32V160c0-17.7-14.3-32-32-32h-84c-6.6 0-12-5.4-12-12V76c0-6.6 5.4-12 12-12h84c53 0 96 43 96 96v192c0 53-43 96-96 96zm-47-201L201 79c-15-15-41-4.5-41 17v96H24c-13.3 0-24 10.7-24 24v96c0 13.3 10.7 24 24 24h136v96c0 21.5 26 32 41 17l168-168c9.3-9.4 9.3-24.6 0-34z"></path>
-              </svg>
-              Sign in
-            </a>
+        <form action="/auth/request">
+          <div class="usa-nav__secondary">
+            <div class="sign-in-wrap clearfix">
+              <input type="submit" class="usa-button usa-button--outline sign-in-bttn float-right" value="Sign in" />
+            </div>
+            <div class="clearfix">
+              <details class="details-popup display-inline-block float-right">
+                <summary class="float-right">Options</summary>
+
+                <label class="usa-label">
```

Comment:
These labels aren't associated with the `<select>` tags. Should either include a `for` attribute, or act as the wrapping container for the `<select>`.

Reply:
Thanks, I remembered to on the SAML app https://github.com/18F/identity-saml-sinatra/pull/71, I'll make a PR to add them here

Reply:
Updated in https://github.com/18F/identity-oidc-sinatra/pull/81